### PR TITLE
Add datetime conversion helper from #18

### PR DESCRIPTION
**Pull request by xjm**
One more small separate scope for review. Soft-blocker for #18.

### CODE REVIEW

File: src/Client/Updater.php [hunk 1]
```diff
@@ -138,6 +139,26 @@ public function validateTarget($targetRepoPath, $targetStream)
...
+     * @param string $timestamp
+     *     The timestamp string in the metadata.
+     *
+     * @return \DateTimeImmutable
+     *     An immutable DateTime object for the given timestamp.
+     *
+     * @throws FormatException
+     *     Thrown if the timestamp string format is not valid.
+     */
+    protected function metadataTimestampToDatetime(string $timestamp) : \DateTimeImmutable
```
Comment (tedbow) [inline src/Client/Updater.php:154]: I think `Datetime` should be `DateTime`  because this is 2 words and how php functions do this.

Comment (xjm) [reply src/Client/Updater.php:154]: Agreed.

Comment (xjm) [reply src/Client/Updater.php]: Pushed a commit to fix this.

**Comment by tedbow** [comment]
Looks good now

**Pull request merged**